### ISSUE TITLE
orm select: buffer overflow, a long string field overwrites other fields nearby

### ISSUE THREAD

**Issue by birdwyx** [vlang/v#15917]
orm select: buffer overflow, a long string field overflows other fields nearby

**V version:** 
V 0.3.1 1ff1f23

**OS:** 
Windows 10 Enterprise LTSC, 21H2

**What did you do?**

tdb.v
```
import mysql


[table: "v_cartoon"]
struct Cartoon {
        id int    //[primary; sql: serial]
pub mut:
        site_id int
        title string
        alias string
        description string
        url string
        cover_url string
        area string
        author string
        status string
        tags string
        last_updated_at string
        create_time int
        update_time int
}


fn main() {

        mut db := mysql.Connection{
                host: 'localhost'
                port: 3306
                username: 'root'
                password: '*****'
                dbname: 'vt'
        }

	db.connect() or {
                panic("error connecting db")
        }

        cartoon := Cartoon{
                id:1008
                site_id:1000
                title: "title"
                alias: "alias"
                description: "looooooooooooooooooooooooooooooooooooooooog description"
                url: "http://url.com"
                cover_url: "http://cover.url.com"
                area: "area"
                author: "author"
                status: "status"
                tags: "tag1, tag2"
                last_updated_at: "2022-9-22"
                create_time: 0 
                update_time: 0     
        }

        sql db {
                insert cartoon into Cartoon 
        }

        cartoon2 := sql db {
                select from Cartoon where id == 1008
        }

        println(cartoon2)

}

```
run the program :
```
v run tdb.v
```

**What did you expect to see?**
```
PS D:\ProjectsMy\vtoon\src> v run tdb.v
Cartoon{
    id: 1008
    site_id: 1000
    title: 'title'
    alias: 'alias'
    description: 'looooooooooooooooooooooooooooooooooooooooog description'
    url: 'http://url.com'
    cover_url: 'http://cover.url.com'
    area: 'area'
    author: 'author'
    status: 'status'
    last_updated_at: '2022-9-22'
    create_time: 0
    update_time: 0
}
```

**What did you see instead?**
```
PS D:\ProjectsMy\vtoon\src> v run tdb.v
Cartoon{
    id: 1008
    site_id: 1953524082
    title: 'oooooooooog description'
    alias: 'oooooooooooooooooooooooooog description'
    description: 'looooooooooooooooooooooooooooooooooooooooog description'
    url: '.com://url.com'
    cover_url: 'http://cover.url.com://url.com'
    area: 'area'
    author: 'author'
    status: 'status'
    tags: 'tag1, tag2'
    last_updated_at: '2022-9-22'
    create_time: 0
    update_time: 0
}
```

NOTE:  the data record in mysql is correct, while orm select gets the wrong

**Comment by enghitalo** [comment]
Same in #15274

**Comment by enghitalo** [comment]
![unknown-5.png](https://user-images.githubusercontent.com/63821277/192978935-40db8218-06fd-46f9-86f0-75c83ff6a2cf.png)

Today, some v orm, mysql is the bad choice, is slow, has more bugs and is a bit more complex to implemented new features

**Comment by walkingdevel** [comment]
Fixed in #18214
